cellular_configs/v1alpha1/config_types.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2023 The Nephio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type ConfigStatus struct {
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+type Config struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
+
+	Spec   ConfigSpec   `json:"spec,omitempty" yaml:"spec,omitempty"`
+	Status ConfigStatus `json:"status,omitempty" yaml:"status,omitempty"`
+}
+
+type ConfigSpec struct {
+	// Config is the embedded config
+	//+kubebuilder:pruning:PreserveUnknownFields
+	ConfigRefs []runtime.RawExtension `json:"config,omitempty" yaml:"config,omitempty"`
```

Comment:
make the name of the tag the same as the Name of the Param. Should configRef

Reply:
Done

Reply:
In fact, I made it `configs` now for both, I think it is better than configRef to avoid any confusion